File: fibonacci/fibonacci.py
# Python program to display the Fibonacci sequence

def fibonacci(n):
   if n <= 1:
       return n
   else:
       return(fibonacci(n-1) + fibonacci(n-2))

N = input()

if N <= 0:
   print("Plese enter a positive integer")
else:
   print("Fibonacci sequence:")
   for i in range(N):
       print(fibonacci(i))

File: fibonacci/fibonacci.cs
cusing System;

namespace Fibonacci
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Enter limit: ");
            calculateFibonacci(Int32.Parse(Console.ReadLine()));
        }

        static void calculateFibonacci (int input)
        {
            int a = 0;
            int b = 1;

            while (b <= input)
            {
                int auxiliar = a; 
                a = b; 
                b = auxiliar + a;
                Console.Write(a + " ");
            }
        }
        
    }
}